$OpenBSD: patch-contrib_groffer_Makefile_sub,v 1.1 2011/03/19 16:48:53 schwarze Exp $
# Do not attempt "install -o root" during the build phase, or you get:
#   install: roff2dvi: chown/chgrp: Operation not permitted
#   Build for all aborted
--- contrib/groffer/Makefile.sub.orig	Fri Jan  9 07:25:52 2009
+++ contrib/groffer/Makefile.sub	Sat Dec 25 13:32:11 2010
@@ -78,9 +78,9 @@ groffer: $(GROFFER_PERL) $(GROFFER_SHELL) \
 	    -e "s|@VERSION@|$(version)$(revision)|g" \
 	    -e "$(SH_SCRIPT_SED_CMD)" \
 	    $(srcdir)/perl/groffer.pl >$@; \
-	  $(INSTALL_DATA) $(srcdir)/perl/groffer.man groffer.man; \
+	  cp $(srcdir)/perl/groffer.man groffer.man; \
 	  for f in $(ROFF2PROGS); do \
-	    $(INSTALL_SCRIPT) $(srcdir)/perl/roff2.pl $$f; \
+	    cp $(srcdir)/perl/roff2.pl $$f; \
 	  done; \
 	else \
 	  : >HAVE_SHELL; \
@@ -92,9 +92,9 @@ groffer: $(GROFFER_PERL) $(GROFFER_SHELL) \
 	    -e "s|@VERSION@|$(version)$(revision)|g" \
 	    -e "$(SH_SCRIPT_SED_CMD)" \
             $(srcdir)/shell/groffer.sh >$@; \
-	  $(INSTALL_DATA) $(srcdir)/shell/groffer.man groffer.man; \
+	  cp $(srcdir)/shell/groffer.man groffer.man; \
 	  for f in $(ROFF2PROGS); do \
-	    $(INSTALL_SCRIPT) $(srcdir)/shell/roff2.sh $$f; \
+	    cp $(srcdir)/shell/roff2.sh $$f; \
 	  done; \
 	fi
 	chmod +x $@
